softgroup/basis/common-router/src/main/java/com/softgroup/common/router/api/RouterHandler_.java

```diff
@@ -0,0 +1,28 @@
+package com.softgroup.common.router.api;
+
+import com.softgroup.common.protocol.Request;
+import com.softgroup.common.protocol.Response;
+import com.softgroup.common.router.api.factory.Factory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class RouterHandler_ implements Handler {
```

Comment:
@AlexBort WTF is RouterHandler with underscore suffix ?? Read Java Code Convention.